src/index.html

```diff
@@ -162,25 +170,45 @@ <h2 class="main-card__title">Question:</h2>
       </section>
     </main>
 
-    <main class="main hidden" data-js="createpage">
-      <section class="main-input">
-        <label for="question">Add question</label>
-        <textarea id="question" name="question"> </textarea>
-      </section>
-      <section class="main-input">
-        <label for="answer">Add answer</label>
-        <textarea id="answer" name="answer"> </textarea>
-      </section>
-      <section class="main-input__small">
-        <label for="tags">Add tags</label>
-        <textarea id="tags" name="tags"> </textarea>
-      </section>
-      <section class="main-submit">
-        <input type="submit" value="Submit" />
-      </section>
+    <main class="main hidden" data-js="create-page">
+      <form action="" data-js="createpage-form">
+        <section class="main-input" data-js="createpage-input">
+          <label for="question">Add question:</label
+          ><span data-js="texarea-maxlength" class="texarea-maxlength"></span>
+          <textarea
+            maxlength="200"
+            data-js="createpage-textarea"
+            id="question"
+            name="question"
+          ></textarea>
+        </section>
+        <section class="main-input" data-js="createpage-input">
+          <label for="answer">Add answer:</label
+          ><span data-js="texarea-maxlength" class="texarea-maxlength"></span>
+          <textarea
+            data-js="createpage-textarea"
+            maxlength="200"
+            id="answer"
+            name="answer"
+          ></textarea>
+        </section>
+        <section class="main-input-small" data-js="createpage-input">
+          <label for="tags">Add tags:</label
+          ><span data-js="texarea-maxlength" class="texarea-maxlength"></span>
+          <textarea
+            data-js="createpage-textarea"
+            maxlength="100"
+            id="tags"
+            name="tags"
+          ></textarea>
+        </section>
+        <section class="main-submit" data-js="submit-button">
```

Comment:
Why an extra section for the button? And why an input of type submit instead of button?